interface/orders/orders_results.php

```diff
@@ -52,13 +52,13 @@
     $result = getPatientData($pid, "*, DATE_FORMAT(DOB,'%Y-%m-%d') as DOB_YMD");
     ?>
   <script language='JavaScript'>
-    parent.left_nav.setPatient(<?php echo js_escape($result['fname'] . " " . $result['lname']) . "," . js_escape($pid) . "," . js_escape($result['pubpid']) . ",''," . js_escape(" " . xl('DOB') . ": " . oeFormatShortDate($result['DOB_YMD']) . " " . xl('Age') . ": " . getPatientAge($result['DOB_YMD'])); ?>);
+    parent.left_nav.setPatient(<?php echo js_escape($result['fname'] . " " . $result['lname']) . "," . js_escape($pid) . "," . js_escape($result['pubpid']) . ",''," . js_escape(" " . xlt('DOB') . ": " . oeFormatShortDate($result['DOB_YMD']) . " " . xlt('Age') . ": " . getPatientAge($result['DOB_YMD'])); ?>);
```

Comment:
We do a different type of escaping in the javascript context (when inside js <script> tags). In this case we use the js_escape() function (which uses json_encode) and we do not do use text/attr/xtt/xla functions in this case. So would revert the above line to its original since already escaped by js_escape() function.

Reply:
Oh okay, got it!  Thank you for the explanation; I understand this revision far better with it!